api/WebStorage.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "WebStorage": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Web_Storage_API",
+        "support": {
+          "webview_android": {
+            "version_added": "2.0"
```

Comment:
Please change to 18. Webview follows Chrome version numbers, not Android version numbers.